src/js/ko-components/view-models/orifice-calculator.coffee

```diff
@@ -62,3 +66,22 @@ class OPL.KoComponents.ViewModels.OrificeCalculator
 
     @compressibilityCorrection = ko.observableArray _.values OPL.OrificeCalculator.Config.Dictionaries.CompressibilityCorrection
     @chosenCompressibilityCorrection  = ko.observable OPL.OrificeCalculator.Config.Dictionaries.CompressibilityCorrection.none
+    @compressibilityCorrectionValue = ko.observable(1)
+    @betaRatio = ko.computed =>
+      @orificeBoreDiameter() / @selectedPipeID()
+
+    @velocityOfApproach = ko.computed =>
+      Math.ceil((1 / Math.sqrt 1 - @betaRatio() ** 4) * 10 ** 2) / 10 ** 2
+
+    @flowRate = ko.computed =>
+      coeffDischarge = 0.6
+      expansionFactor = 1
+      baseTemperature = 519.67 # Tb in Rankine, assumed 60F
+      # TODO: make observable return integer instead of string by default?
+      operatingTemperatureInRankine = Number(@operatingTemperature()) + 459.67
```

Comment:
The 459.67 constant im pretty sure it has a name and could be moved to the constants as well.

All the other constants should be moved